stagemonitor-core/src/main/java/org/stagemonitor/core/elasticsearch/AbstractElasticsearchInitializer.java

```diff
@@ -13,9 +13,6 @@ public void init(ConfigurationRegistry configurationRegistry) {
 	}
 
 	public final void onElasticsearchAvailable(ElasticsearchClient elasticsearchClient) {
-		if (!corePlugin.isInitializeElasticsearch()) {
```

Comment:
Removing this line will lead to the execution of the other `ElasticsearchInitializer`, e.g. `OsElasticsearchInitializer`. They will try to create dashboards - and this should not happen if `stagemonitor.elasticsearch.init=false`. The creation inside the other initializers should be guarded by `if (elasticsearchClient.isElasticsearch6Compatible()) { ... }` as well.

Reply:
All `AbstractElasticsearchInitializer` implementations use the same `sendMetricDashboardBulkAsync` method in `ElasticsearchClient`. In my opinion we should do the check there.